docs/debug.md

```diff
@@ -0,0 +1,196 @@
+
+* Fetch from remote registry
+    * **operation_duration_remote_registry_get (ms)** - measures the time it takes to complete a `GET` operation from remote registry for a specific layer. This metric should help in identifying network issues, when lazily fetching layer data and seeing increased container start time.
+* FUSE
+    * **operation_duration_node_readdir (us)** - measures the time it takes to complete readdir() operation for a file from a specific layer. The per-layer granularity is to point out that each layer has its own `FUSE` mount, so it doesn’t make sense to generalize. The unit is microseconds. Large times in readdir may indicate that there are problems with the request speed from metadata db or issues with the `FUSE` implementation (less likely, since this part is least likely to get modified).
+    * **operation_duration_synchronous_read (us)** - measures the duration of `FUSE` read() operation for the specific `FUSE` mountpoint, defined by the layer digest. The unit of measurement is microseconds.
+    * **synchronous_read_count** - measures how many read() operations were issued for the specific `FUSE` mountpoint (defined by the layer digest) to date. The  same value can be obtained from `operation_duration_synchronous_read` as the Count property.
+    * **synchronous_bytes_served** - measures the number of bytes served for synchronous reads. 
+    * **fuse_mount_failure_count** - number of times the snapshotter falls back to use a normal overlay mount instead of mounting the layer as a `FUSE` mount.
+    * **background_span_fetch_failure_count** - number of errors of span fetch by background fetcher.
+    * **background_span_fetch_count** - number of spans fetched by background fetcher.
+    * **background_fetch_work_queue_size** - number of items in the work queue of background fetcher.
+    * **operation_duration_background_fetch** - time in milliseconds to complete background fetch for a layer.
+    * Individual `FUSE` operation failure counts:
+      * fuse_node_getattr_failure_count
+      * fuse_node_listxattr_failure_count
+      * fuse_node_lookup_failure_count
+      * fuse_node_open_failure_count
+      * fuse_node_readdir_failure_count
+      * fuse_file_read_failure_count
+      * fuse_file_getattr_failure_count
+      * fuse_whiteout_getattr_failure_count
+      * fuse_unknown_operation_failure_count
+
+# Common Scenarios
+
+Below are some common scenarios that may occur during `rpull` and the lifetime of running a container. For scenarios not covered, please feel free to [open an issue](https://github.com/awslabs/soci-snapshotter/issues/new/choose).
+
+> _Note: To allow for more verbose logging you can set the `--log-level` flag to `debug` when starting the snapshotter._
```

Comment:
Can we use Github Markdown Note syntax? https://github.com/community/community/discussions/16925